docs/usage/self-hosted-configuration.md

```diff
@@ -266,11 +266,11 @@ If found, it will be imported into `config.npmrc` with `config.npmrcMerge` set t
 
 The format of the environment variables must follow:
 
-- Datasource name (e.g. `NPM`, `PYPI`)
+- Datasource name (e.g. `NPM`, `PYPI`) or Platform name (e.g. `GITHUB`)
```

Comment:
We need to test thoroughly that this doesn't cause unintended logs or errors if variables like GITHUB_TOKEN are in use

Reply:
sure. I have made a change and make a unit test for it.

Reply:
do you think the change I made is sufficient to cover the situation (e.g. GITHUB_TOKEN) you mentioned?  @rarkins . let me know if there other things I could help

Reply:
Do you mean this line? `(platforms.has(hostType) && splitEnv.length > 1)`

i.e. it needs to be `GITHUB_X_TOKEN` or longer?

Reply:
I was intended to just to avoid processing environment variable like `{{PLATFORM}}_TOKEN`. would you like to just skip those environment variable from a list ?

Reply:
I think it's ok to skip all of them, although best to document that fact

Reply:
cool. I've updated the doc.